tests/integration/test_render.py

```diff
@@ -194,3 +202,21 @@ def test_background_image(self):
         image_path = '/resource/drag-and-drop-v2/public/img/triangle.png'
         self.assertTrue(bg_image.get_attribute("src").endswith(image_path))
         self.assertEqual(bg_image.get_attribute("alt"), 'This describes the target image')
+
+    def test_borders_inactive(self):
+        self.load_scenario()
+        zones = self._get_zones()
+        for index, zone in enumerate(zones):
+            elt = 'zone-{}'.format(index + 1)
+            # Firefox does not provide a single "border-style", so we check "border-top-style"
+            self.assertEqual(self._get_border_style(elt, 'border-top-style'), 'none')
+            self.assertEqual(self._get_border_style(elt, 'border-top-width'), '0px')
+
+    def test_borders_active(self):
+        self.load_scenario("", "", True)
```

Comment:
@agaylard Since all parameters of `load_scenario` have default values (i.e., they are keyword arguments), you don't have to pass values for `item_background_color` and `item_text_color` here. Just do:

```python
self.load_scenario(borders=True)
```